Datasets_for_Final_Project.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Summary of health datasets to use for the final project
```

Comment:
This should be the first line, everything above it must be removed

Reply:
Removed. Thanks.